components/script/bluetooth_blacklist.rs

```diff
@@ -5,115 +5,123 @@
 use regex::Regex;
 use std::cell::RefCell;
 use std::collections::HashMap;
-use std::fs::File;
-use std::io::{BufRead, BufReader};
 use std::string::String;
+use util::resource_files::read_resource_file_lines;
 
 const EXCLUDE_READS: &'static str = "exclude-reads";
 const EXCLUDE_WRITES: &'static str = "exclude-writes";
-const FILE_PATH: &'static str = "./resources/gatt_blacklist.txt";
 const VALID_UUID_REGEX: &'static str = "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}";
 
 thread_local!(pub static BLUETOOTH_BLACKLIST: RefCell<BluetoothBlacklist> =
               RefCell::new(BluetoothBlacklist(parse_blacklist())));
 
-macro_rules! return_if_blacklisted {
-    ($uuid:expr, $exclude_fn:ident) => {
-        if BLUETOOTH_BLACKLIST.with(|blist| {
-            blist.borrow().$exclude_fn($uuid.as_ref())
-        }) {
-            return Err(Security);
-        }
+pub fn uuid_is_blacklisted(uuid: &str, exclude_type: Blacklist) -> bool {
+    match exclude_type {
+        Blacklist::All => {
+            BLUETOOTH_BLACKLIST.with(|blist| {
```

Comment:
The following code seems to be repeatedly used:
```
BLUETOOTH_BLACKLIST.with(|blist| {
    blist.borrow().is_blacklisted*
})
```
can we somehow refactor this?